data/campaigns/Under_the_Burning_Suns/scenarios/04_Descending_into_Darkness.cfg

```diff
@@ -603,6 +616,21 @@
             speaker=Kaleh
             message= _ "How can he move that fast? He is faster than any goblin rider I have ever seen. It’s almost unnatural."
         [/message]
+
+        [if]
+            [variable]
+                name=sneak_up
+                boolean_equals=yes
+            [/variable]
+            [then]
+                [message]
+                    speaker=Nym
+                    message= _ "Spotted immediately! Well, I guess I spoke too soon about trying sneaking."
```

Comment:
"trying sneaking" isn't technically wrong I think, but it sounds weird to me. I'd go with "trying to sneak" instead.